pkg/components/helpers.go

```diff
@@ -135,3 +137,31 @@ func handleUpdatingClusterState(
 	}
 	return nil, err
 }
+
+func SetPathAcl(path string, acl []yt.ACE) string {
+	aclAsString, err := yson.MarshalFormat(acl, yson.FormatText)
```

Comment:
I suppose this shouldn't be called `*asString` since it is bytes.

Reply:
Ok, I will rename it to `formattedAcl`.